feconf.py

```diff
@@ -1449,3 +1449,6 @@ def get_empty_ratings():
     SUGGESTION_TYPE_TRANSLATE_CONTENT,
     SUGGESTION_TYPE_ADD_QUESTION
 ]
+
+# Regex to match entity id's.
+REGEX_FOR_ENTITY_ID = r'^[a-zA-Z0-9\-_]{1,12}$'
```

Comment:
Can you move this closer to the regex for URL fragment?

Reply:
Added this regex in constants.ts in a close proximity with other regex's.